ceremonies/src/lib.rs

```diff
@@ -67,11 +67,8 @@ pub mod pallet {
 	use frame_support::pallet_prelude::*;
 	use frame_system::pallet_prelude::*;
 
-	const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
```

Comment:
Why did you remove this? We need this as we implemented storage migration lately.

Reply:
Fixed it [4dc5183](https://github.com/encointer/pallets/pull/322/commits/4dc518340aadcdb64148393ff27ac6ee526df993).